docs/pages/agents/join-services-to-your-cluster/aws-iam.mdx

```diff
@@ -3,22 +3,43 @@ title: Joining Services via AWS IAM Role
 description: Use the IAM join method to add services to your Teleport cluster on AWS
 ---
 
-This guide will explain how to use the **IAM join method** to configure Teleport
+This guide explains how to use the **IAM join method** to configure Teleport
 processes to join your Teleport cluster without sharing any secrets when they
 are running in AWS.
 
+There are two other AWS join methods available depending on your use
+case:
+
+1. The **EC2 join method**, which is only available for self-hosted Teleport
+   deployments (see [documentation](aws-ec2.mdx). A Teleport process running on
+   an EC2 instance requests a signed EC2 instance identity document and presents
+   it to the Teleport Auth Service, which queries the EC2 API to verify that the
+   instance is genuine. This method requires a self-hosted Teleport cluster and
+   IAM credentials for the Teleport Auth Service.
+
+1. **Tokens not signed by AWS:** You can configure Teleport processes running on
+   AWS to join a cluster via [Teleport join tokens](join-token.mdx) or, for
+   Teleport processes running on Kubernetes, [signed ServiceAccount
+   tokens](kubernetes.mdx). These approaches allow you to join a Teleport
+   process to a cluster when you don't want to rely on AWS-specific APIs, e.g.,
+   when adopting a cloud-agnostic approach.
+
+## How it works
+
 The IAM join method is available to any Teleport process running anywhere with
 access to IAM credentials, such as an EC2 instance with an attached IAM role.
 No specific permissions or IAM policy is required: an IAM role with no attached
 policies is sufficient. No IAM credentials are required on the Teleport Auth
 Service.
 
+Teleport processes prove that they are running in your AWS account by sending a
+pre-signed `sts:GetCallerIdentity` request to the Teleport Auth Service. The
+service's identity must match an allow rule configured in your AWS service
+joining token.
```

Comment:
not sure about "AWS service joining token", is that a term we use? oh you just moved this, I probably wrote it 😅 Maybe just "join token" is better